Source/Rio.Web/src/app/pages/parcel-ledger-bulk-create/parcel-ledger-bulk-create.component.ts

```diff
@@ -71,7 +72,7 @@ export class ParcelLedgerBulkCreateComponent implements OnInit {
         this.parcelService.getParcelAllocationAndUsagesByYear(new Date().getFullYear())
       ).subscribe(([waterTypes, parcelAllocationAndUsagesByYear]) => {
         this.waterTypes = waterTypes.filter(x => x.IsUserDefined);
-        this.rowData = parcelAllocationAndUsagesByYear;
+        this.parcelAllocationAndUsagesByYear = parcelAllocationAndUsagesByYear;
```

Comment:
you're handling the eliminating of the word allocation on the other story?

Reply:
Yeah, that'll be on Rio 355